Blender2Helios.py

```diff
@@ -36,7 +36,7 @@
 class Blender2HeliosPreferences(AddonPreferences):
     bl_idname = __name__
     
-    pref_heliosBaseDir: StringProperty(name="Helios Base Directory", description="Directory containing 'assets' and 'data' directories", subtype='DIR_PATH', default=expanduser("~")+"/")
+    pref_heliosBaseDir: StringProperty(name="Helios Base Directory", description="Directory containing 'assets' and 'data' directories", subtype='DIR_PATH', default=os.getcwd())
```

Comment:
The Blender file chooser always adds a separator ("/" or "\") at the end of directories. This is expected in other functions. os.getcwd() doesn't do this.
I will add some more code.